packages/frontend/app/controllers/reports/subject.js

```diff
@@ -6,6 +6,7 @@ export default class ReportsSubjectController extends Controller {
   @service store;
 
   queryParams = ['report', 'reportYear'];
+  resultsLengthMax = 200;
```

Comment:
Since each report results display has to manage this it makes more sense to me to have it be a value within that component and not something that gets passed down. Maybe make it a constant for easier search / readability, but I don't see us wanting to make this configurable or anything special so let's drop the mental overhead of passing it around.